backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/Importer.kt

```diff
@@ -0,0 +1,25 @@
+package app.ehrenamtskarte.backend.stores.importer
+
+import app.ehrenamtskarte.backend.config.BackendConfiguration
+import app.ehrenamtskarte.backend.stores.importer.pipelines.EhrenamtskarteBayern
+import app.ehrenamtskarte.backend.stores.importer.pipelines.SozialpassNuernberg
+import org.slf4j.LoggerFactory
+
+object Importer {
+    fun import(config: BackendConfiguration): Boolean {
+        val logger = LoggerFactory.getLogger(Importer::class.java)
+
+        return try {
+            when (config.project.pipelineName) {
+                "EhrenamtskarteBayern" -> EhrenamtskarteBayern.import(config, logger)
+                "SozialpassNuernberg" -> SozialpassNuernberg.import(config, logger)
+                else -> throw Error("Invalid pipeline name '${config.project.pipelineName}'!")
+            }
```

Comment:
Better ideas or other preferences? I tried and thought about reflection but wasn't that nice/beautiful either.

Reply:
looks good!